qmxgraph/api.py

```diff
@@ -685,10 +685,47 @@ def restore(self, state):
         """
         Restore the graph's state to one saved with `dump`.
 
-        :param str state: A xml string previously obtained with `bump`.
+        :param str state: A xml string previously obtained with `dump`.
         """
         return self.call_api('restore', state)
 
+    def set_cells_deletable(self, enabled):
+        self._call_graph_api('setCellsDeletable', enabled)
+
+    def is_cells_deletable(self):
+        return self._call_graph_api('isCellsDeletable')
+
+    def set_cells_disconnectable(self, enabled):
+        self._call_graph_api('setCellsDisconnectable', enabled)
+
+    def is_cells_disconnectable(self):
+        return self._call_graph_api('isCellsDisconnectable')
+
+    def set_cells_editable(self, enabled):
+        self._call_graph_api('setCellsEditable', enabled)
+
+    def is_cells_editable(self):
+        return self._call_graph_api('isCellsEditable')
+
+    def set_cells_movable(self, enabled):
+        self._call_graph_api('setCellsMovable', enabled)
+
+    def is_cells_movable(self):
+        return self._call_graph_api('isCellsMovable')
+
+    def set_connectable(self, enabled):
+        self._call_graph_api('setConnectable', enabled)
+
+    def is_connectable(self):
+        return self._call_graph_api('isConnectable')
+
+    def _call_graph_api(self, fn, *args):
```

Comment:
I have somn mixed feelings about this and allowing fow the python api to diverge from the javascript api.

Reply:
@nicoddemus , any suggestions?

Reply:
Agree, as we discussed @prusse-martin!